app/forms/waste_exemptions_engine/registration_complete_form.rb

```diff
@@ -18,5 +18,9 @@ def initialize(registration)
     def submit(_params)
       raise UnsubmittableForm
     end
+
+    def self.can_navigate_flexibly?
```

Comment:
I think the intent was to add this to the base form and default to `true` but then override it where we need to, for example here and on the declaration form.

Reply:
@Cruikshanks That's already in the BaseForm! I thought it had been removed but looks like not.

Reply:
And that's why I set this as a comment 🤦‍♂ Thx @irisfaraway and ignore me @cintamani

Reply:
https://github.com/DEFRA/waste-exemptions-engine/pull/93 :D